docs/source/config_file.rst

```diff
@@ -418,7 +419,7 @@ section of the command line docs.
     variable.
 
     Note that the cache is only read when incremental mode is enabled
-    but is always written to, unless the value is set to ``/dev/nul``
+    but is always written to, unless the value is set to ``/dev/null``
```

Comment:
This isn't related to the crossreferencing (and it's the only one unrelated change I included in this PR), but is a simple typo, so I decided to include it.

Reply:
+1